CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## [TBD] - TBD
+## [1.0.1] - 2020-10-14
```

Comment:
Leave the date as "TBD" as we're probably not going to release it now, we'll only do it once we reach a milestone in the series of changes you're making.